src/parser/types.rs

```diff
@@ -1133,6 +1133,18 @@ impl FunctionDef {
     pub fn parse_fn_body(ctx: &mut ParserCtx, tokens: &mut Tokens) -> Result<Vec<Stmt>> {
         let mut body = vec![];
 
+        // return empty body when the next token is `;` instead of `{`
```

Comment:
What is the scenario for that? Do we have test cases?

Reply:
Before this PR, the hint functions are implemented in builtins. So the case of empty body will be deprecated.